qmaftools/src/org/qcmg/maf/util/MafUtils.java

```diff
@@ -445,13 +428,13 @@ public static void loadDCCFile(String dccqFile, Map<ChrPosition, TorrentVerifica
 	}
 	
 	
-	public static void convertDccToMaf(TabbedRecord tabbedRecord, String patientId, String controlSampleID, 
+	public static void convertDccToMaf(String String, String patientId, String controlSampleID, 
```

Comment:
surely we can't have a variable called `String`!

Reply:
yes, it was typo, fixed